uQR.py

```diff
@@ -960,7 +962,7 @@ def create_data(version, error_correction, data_list):
 def make(data=None, **kwargs):
     qr = QRCode(**kwargs)
     qr.add_data(data)
-    return qr.make_image()
+    return qr.get_matrix()
```

Comment:
Hmm....it looks like the `make` method wasn't under test or documentation. I'd like to consider removing it rather than maintaining it. If maintaining it, I'd prefer to put it under test.

What are your thoughts?

Reply:
Happy to see it removed, although it's invoked in some documentation examples, the implementation is trivial enough to not be of much utility.